### ISSUE TITLE
Properly implement the gas station

### ISSUE THREAD

**Issue by FamTroder** [FamTroder/FamMel#62]

**Comment by M0nkey5ee** [comment]
What do you have in mind with "properly"? Maybe enhancing it with some elements inspired by the diane-game? I like the sound of it, I just don't have any vision for it right now. But could even add a scene where she barely makes it out of the car and pees right then and there? One where she gets the key to the toilet but cannot hold it until she reaches it? Sounds great either way!

**Comment by Forget-About-Me2** [comment]
I don't have a concrete plan yet, but I did came across what I wrote years back on ideas on this so adding this here for consideration:

-  The idea is in the end that you have to pay for the toilet there and she might ask you for money which you can refuse, I'm also planning to actually give her some money herself, which you can steal from her purse.

- I'm also planning to have her ask for the gas station and if you don't actually stop you might lose attraction or something

**Comment by M0nkey5ee** [comment]
I personally am not a big fan of stealing options all together, especially when she is into stuff like this (which would even make sense as at least the final freedom apparently feels like heaven for most women). But I think there is a lot of potential with the gas station alone, there I definitly agree!

**Comment by Forget-About-Me2** [comment]
Yeah I'm not really a fan of stealing options either and some similar things, but they have been in the game for a long time, implemented by one of the previous developers so I am not going to take it out and if I give her money and you can't steal it while you can steal her other stuff feels weird I guess.
